bee-crypto/src/ternary/hash.rs

```diff
@@ -0,0 +1,78 @@
+// Copyright 2020 IOTA Stiftung
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+// the License. You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+// an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and limitations under the License.
+
+use bee_ternary::{Trits, T1B1};
+
+use std::{cmp::PartialEq, fmt, hash};
+
+/// The length of a hash in units of balanced trits.
+pub const HASH_LENGTH: usize = 243;
+
+/// Ternary cryptographic hash.
+#[derive(Copy, Clone)]
+pub struct Hash([i8; HASH_LENGTH]);
+
+impl Hash {
+    /// Creates a new `Hash` from trits.
+    pub fn new(trits: [i8; HASH_LENGTH]) -> Self {
+        Self(trits)
+    }
```

Comment:
Is this the only way to create a Hash? Or maybe there could be a From trait for Trits type in bee-ternary?

Reply:
A new ctor will be added.